.github/workflows/rust.yml

```
@@ -40,64 +38,44 @@
-        run: cargo test
-      - name: Run tests for std feature
-        run: cargo test --features std
-      - name: Run tests for extensions
-        run: cargo test --features extensions --lib extensions -- --test-threads=1
-      - name: Run doc tests
-        run: cargo test --doc --all-features
+        run: cargo +${{ matrix.rust-version }} build --features extensions
+      - name: Build with all features
+        run: cargo +${{ matrix.rust-version }} build --all-features
+      - name: Run tests
+        if: matrix.test-features
```

actionlint (1.7.7)

property "test-features" is not defined in object type {rust-version: string}

(expression)